testsuites/CBLTester/System_test_multiple_device/conftest.py

```diff
@@ -358,7 +369,8 @@ def params_from_base_suite_setup(request):
         "create_db_per_test": create_db_per_test,
         "enable_rebalance": enable_rebalance,
         "enable_encryption": enable_encryption,
-        "encryption_password": encryption_password
+        "encryption_password": encryption_password,
+        "delta_sync_enabled": delta_sync_enabled
```

Comment:
do you need to pass the delta_sync_enabled parameter to test case level?

Reply:
Not now, but if someone need the flag to skip a test or to do some other check, they would need it

Reply:
that's okay for  now, I would suggest not to add unless it is required for the existing tests. We should add as and when required.